jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -804,6 +805,12 @@ protected void cleanupAfterStop() throws Exception
         }
     }
 
+    /** Generate a reasonable name for the temp directory because one has not be
+     * explicitly configured by the user. The directory may also be created, if
+     * it is not persistent. If it is persistent it will be created as necessary by
+     * {@link #createTempDirectory()} later during the startup of the context.
+     * @throws Exception
+     */
```

Comment:
The Javadoc needs an {@link or @see to setTempDirectory to better explain the relationship

Reply:
Done.